package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "2.1.8",
+  "version": "3.0.0",
```

Comment:
No change to `vue-test-utils-helps` spec on line 84?

Reply:
i changed the package-lock.json directly to fix the +ssh to +https   as recommended by @pwei1018

Reply:
Obsolete comment.